src/components/course/data/utils.jsx

```diff
@@ -633,8 +662,8 @@ export const getMissingSubsidyReasonActions = ({
         size="sm"
         block
       >
-        Contact administrator
-        <span className="sr-only">about funds</span>
+        {noFundsButtonText}
+        <span className="sr-only">{noFundsButtonText}</span>
```

Comment:
I believe we may want to follow the existing convention here, where the button text says "Contact administrator" but the `sr-only` text reads out "for help".

As is, a screen reader like voiceover would read out "Contact admin for help" twice: once for the displayed button text, and again for the `sr-only` text. Ideally, it would read a singular "Contact administrator [for help]".

Reply:
Thanks, didn't know about that convention.